### DISCUSSION TITLE
Allow to select scheme or favours in `cap run`

### DISCUSSION THREAD

**Discussion by MortenGregersen** [ionic-team/capacitor#3924] [Ideas]
It looks fantastic with the plans and all the work that goes into Capacitor 3!

I see that the `cap run` has been added - awesome! We would like to use this, but as we are working on a white label app and have defined multiple schemes in Xcode and flavours in Android Studio, we need a way to specify this when running `cap run`.

I think this could just be a parameter to the command.

**Comment by imhoffd** [comment]
Hi @MortenGregersen! I created https://github.com/ionic-team/capacitor/issues/3993 to track this.
